### PR TITLE
document for add new post

### PR DESCRIPTION
**Pull request by mvn-caonguyen-dn**
## Description
Document for add new post
## Reference Requirements
- Product Backlog ID: PBI-40
- Task ID: #40.1

## Reference Isuees
#{issue_id}

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

### CODE REVIEW

File: docs/api-references/post.md [hunk 1]
```diff
@@ -423,3 +423,74 @@ Get list all posts by user with paginate
...
+            "content": "nguyen vancao",
+            "status": 2,
+            "name": "Cao Nguyen V.",
+            "team": "PHP",
+            "avatar_url": "http://172.16.110.17/images/user/avatar/366/64314e61c9.png",
+            "is_admin": 0,
+            "picture": "http://book.aug/images/books/20180209.jpeg",
+            "title": "Prof. Julio Bechtelar III",
+            "book_id": 3,
+            "rating": "3.0",
```
Comment (at-nhannguyen2) [inline docs/api-references/post.md]: chuyển về định dạng kiểu number

**Review by ghost — COMMENTED** [review]
check cmt

**Review by ghost — COMMENTED** [review]
LGTM

**Comment by at-huydq** [comment]
LGTM

**Pull request merged**